thunder/core/dtypes.py

```diff
@@ -147,6 +152,10 @@ def __init__(self, name, shortname, *, bytes, is_weak):
 uint64 = unsignedinteger("uint", "ui", bytes=8, is_weak=False)
 uint64_ = unsignedinteger("uint", "ui", bytes=8, is_weak=True)
 
+# TODO: Update _elementwise_exact_promotion_table to handle uint64 and uint64_
+all_dtypes.discard(uint64)
+all_dtypes.discard(uint64_)
```

Comment:
This dtype wasn't part of `all_dtypes` before this PR and there are type promotion tables to be updated to support this in testing.